packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -66,6 +66,9 @@ library TaikoData {
         bytes signature;
         uint32 l1StateBlockNumber;
         uint64 timestamp;
+        uint256 blobTxListOffset;
+        uint256 blobTxListLength;
```

Comment:
Also add these two to `BlockMetadata` because the prover will need to read the slice info from there.

Reply:
kk